tasks/restore.yml

```diff
@@ -1,23 +1,47 @@
 ---
+- block:
+    - name: Sync etcd backup file {{ restore_file_name }} from localhost to all etcd nodes
+      synchronize:
+        use_ssh_args: yes
+        src: "{{ restore_file_path }}/{{ restore_file_name }}"
+        dest: "{{ etcd_backup_local_root_dir }}"
+    
+    - name: Fix backup file {{ restore_file_name }} ownership on all etcd nodes
+      file:
+        path: "{{ etcd_backup_local_root_dir }}/{{ restore_file_name }}"
+        owner: etcd
+        group: root
+  become: yes
+  when: restore_file_name is defined and restore_file_path is defined
+
+- name: Set etcd_backup_latest_file fact if using file option
```

Comment:
Personally, I like this name better:
"Set etcd_backup_latest_file fact if the user-defined etcd backup file is provided"